datafusion-examples/README.md

```diff
@@ -35,6 +35,9 @@ cd datafusion
 # Download test data
 git submodule update --init
 
```

Comment:
I tested this locally and the existing command (in the main datafusion directory) also seems to work fine

```
~/Software/datafusion2$ cargo run --example csv_sql

     Running `target/debug/examples/csv_sql`
+----+-----------------------------+-----------------------------+
| c1 | MIN(aggregate_test_100.c12) | MAX(aggregate_test_100.c12) |
+----+-----------------------------+-----------------------------+
| c  | 0.2667177795079635          | 0.991517828651004           |
| a  | 0.02182578039211991         | 0.9567595541247681          |
| b  | 0.04893135681998029         | 0.9185813970744787          |
| e  | 0.01479305307777301         | 0.9965400387585364          |
| d  | 0.061029375346466685        | 0.9748360509016578          |
+----+-----------------------------+-----------------------------+
+----+------+
| c2 | c3   |
+----+------+
| 1  | -85  |
| 3  | 13   |
| 4  | -38  |
| 4  | -54  |
| 5  | 36   |
| 1  | -25  |
| 5  | -31  |
| 2  | 45   |
| 3  | 13   |
| 3  | 17   |
| 4  | 65   |
| 4  | -101 |
| 2  | -48  |
| 1  | -56  |
| 1  | -5   |
| 3  | 14   |
| 1  | 83   |
| 3  | -12  |
| 3  | -72  |
| 2  | -43  |
| 5  | -101 |
+----+------+
```

It does also work in `datafusion-examples/examples`/ but `cd`ing into that directory seems unecessary

Reply:
thanks @alamb for testing this.

let me quote the commit message, maybe it explains my reasoning better:

```
...
Some examples are runnable from any place (e.g. `csv_sql`), but some
expect a specific working directory (e.g. `regexp`). Running from
`datafusion-examples/examples` is tested on CI so guaranteed to work,
let's put this path in the README.
...
```

> It does also work in datafusion-examples/examples/ but cding into that directory seems unecessary

i agree.
It happened to be necessary for the first example i tried to run. maybe i was just unlucky

per the commit message

```
...
As a follow-up, we should look what it would take to make examples
runnable directly from an IDE such as RustRover.
```

-- I also am against cd-ing into directory as unnecessary and also want the examples to be runnable from an IDE.
If we want this to be the case, this validated by the CI scripts.